users/api/metrics.go

```diff
@@ -100,3 +100,76 @@ func (ms *metricsMiddleware) SendPasswordReset(ctx context.Context, host, email,
 
 	return ms.svc.SendPasswordReset(ctx, host, email, token)
 }
+
+func (ms *metricsMiddleware) CreateGroup(ctx context.Context, token string, group users.Group) (users.Group, error) {
+	defer func(begin time.Time) {
+		ms.counter.With("method", "create_group").Add(1)
+		ms.latency.With("method", "create_group").Observe(time.Since(begin).Seconds())
+	}(time.Now())
+
+	return ms.svc.CreateGroup(ctx, token, group)
+}
+
+func (ms *metricsMiddleware) ListGroups(ctx context.Context, token, id string, offset, limit uint64, meta users.Metadata) (users.GroupPage, error) {
+	defer func(begin time.Time) {
+		ms.counter.With("method", "list_groups").Add(1)
+		ms.latency.With("method", "list_groups").Observe(time.Since(begin).Seconds())
+	}(time.Now())
+
+	return ms.svc.ListGroups(ctx, token, id, offset, limit, meta)
+}
+
+func (ms *metricsMiddleware) ListUsersInGroup(ctx context.Context, token, id string, offset, limit uint64, meta users.Metadata) (users.UserPage, error) {
```

Comment:
This could be `ListGroupUsers`

Reply:
@manuio `ListUsersGroup`  doesn't it sound more logical? Following work orders  `List` `Users` `Group` e.g List all users group. Detail ;)

Reply:
@nmarcetic I don't think so :) We want users that are in a group not the group belonging to users.

Reply:
Hm @manuio Yea, that makes more sense :)